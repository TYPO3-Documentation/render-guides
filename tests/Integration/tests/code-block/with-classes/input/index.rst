============
With classes
============

..  code-block:: python
    :class: some-class another-class

    def some_function():
        interesting = False
