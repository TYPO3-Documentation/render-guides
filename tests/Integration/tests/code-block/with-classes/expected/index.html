<!-- content start -->
                <section class="section" id="with-classes">
            <h1>With classes<a class="headerlink" href="#with-classes" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">Â¶</a></h1>
            <div class="code-block-wrapper" translate="no">

    <pre class="code-block some-class another-class"><code class="hljs python"><span class="hljs-function"><span class="hljs-keyword">def</span> <span class="hljs-title">some_function</span><span class="hljs-params">()</span>:</span>
    interesting = <span class="hljs-literal">False</span></code></pre>
    <div>        <button class="code-block-copy" title="Copy to clipboard">
            <i class="icon fa-regular fa-copy code-block-copy-icon"></i>
            <i class="fa-solid fa-check code-block-check-icon code-block-hide"></i>
        </button>
        <span class="code-block-check-tooltip code-block-hide" aria-live="polite">Copied!</span>
    </div>
</div>
    </section>
        <!-- content end -->
